terraform/postgres.tf

```diff
@@ -0,0 +1,22 @@
+# Get data about the postgres service on the PaaS
+# so we can get the guid of the service plan we want to use.
+data "cloudfoundry_service" "postgres" {
+  name = "postgres"
+}
+
+# Create a postgres database named with the environment.
+# Enable the pgcrypto and plpgsql extensions.
+# Increase the timeouts since the default of 15 minutes is too
+# short for postgres db creation
+
+resource "cloudfoundry_service_instance" "beis-roda-postgres" {
+  name         = "beis-roda-${var.environment}-postgres"
+  space        = cloudfoundry_space.space.id
+  service_plan = data.cloudfoundry_service.postgres.service_plans["small-ha-11"]
```

Comment:
I wonder if the plan size should now be 100GB (with `medium-ha-11`) instead of 20GB from the off as we are likely to forget about this and have to deal with it quite quickly in live when we start backfilling years worth of transactions.

Reply:
sure.

Reply:
I think this should be another PR which gets rolled carefully when we are ready for it. While Adam is currently adding data is probably not the best time.

Reply:
Agree. This ship has sailed :)